File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SCH3001U00GrdSCH3000Info.java
package nta.med.data.model.ihis.schs;

import java.sql.Timestamp;

public class SCH3001U00GrdSCH3000Info {
	private Timestamp jukyongDate;
    private String jundalTable;
    private String jundalPart;
    private String gumsaja;
    private String yoilGubun; 
    private String startTime;
    private String endTime;
    private Double iwon;
    private Double addIwon;
    private Double outHospSlot;
	public SCH3001U00GrdSCH3000Info(Timestamp jukyongDate, String jundalTable,
			String jundalPart, String gumsaja, String yoilGubun,
			String startTime, String endTime, Double iwon, Double addIwon, Double outHospSlot) {
		super();
		this.jukyongDate = jukyongDate;
		this.jundalTable = jundalTable;
		this.jundalPart = jundalPart;
		this.gumsaja = gumsaja;
		this.yoilGubun = yoilGubun;
		this.startTime = startTime;
		this.endTime = endTime;
		this.iwon = iwon;
		this.addIwon = addIwon;
		this.outHospSlot = outHospSlot;
	}
	public Timestamp getJukyongDate() {
		return jukyongDate;
	}
	public void setJukyongDate(Timestamp jukyongDate) {
		this.jukyongDate = jukyongDate;
	}
	public String getJundalTable() {
		return jundalTable;
	}
	public void setJundalTable(String jundalTable) {
		this.jundalTable = jundalTable;
	}
	public String getJundalPart() {
		return jundalPart;
	}
	public void setJundalPart(String jundalPart) {
		this.jundalPart = jundalPart;
	}
	public String getGumsaja() {
		return gumsaja;
	}
	public void setGumsaja(String gumsaja) {
		this.gumsaja = gumsaja;
	}
	public String getYoilGubun() {
		return yoilGubun;
	}
	public void setYoilGubun(String yoilGubun) {
		this.yoilGubun = yoilGubun;
	}
	public String getStartTime() {
		return startTime;
	}
	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}
	public String getEndTime() {
		return endTime;
	}
	public void setEndTime(String endTime) {
		this.endTime = endTime;
	}
	public Double getIwon() {
		return iwon;
	}
	public void setIwon(Double iwon) {
		this.iwon = iwon;
	}
	public Double getAddIwon() {
		return addIwon;
	}
	public void setAddIwon(Double addIwon) {
		this.addIwon = addIwon;
	}

	public Double getOutHospSlot() {
		return outHospSlot;
	}

	public void setOutHospSlot(Double outHospSlot) {
		this.outHospSlot = outHospSlot;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SCH3001U00GrdSCH3000Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
    public class SCH3001U00GrdSCH3000Info
    {
        private String _jukyongDate;
        private String _jundalTable;
        private String _jundalPart;
        private String _gumsaja;
        private String _yoilGubun;
        private String _startTime;
        private String _endTime;
        private String _iwon;
        private String _addIwon;
        private String _outHospSlot;
        private String _dataRowState;

        public String JukyongDate
        {
            get { return this._jukyongDate; }
            set { this._jukyongDate = value; }
        }

        public String JundalTable
        {
            get { return this._jundalTable; }
            set { this._jundalTable = value; }
        }

        public String JundalPart
        {
            get { return this._jundalPart; }
            set { this._jundalPart = value; }
        }

        public String Gumsaja
        {
            get { return this._gumsaja; }
            set { this._gumsaja = value; }
        }

        public String YoilGubun
        {
            get { return this._yoilGubun; }
            set { this._yoilGubun = value; }
        }

        public String StartTime
        {
            get { return this._startTime; }
            set { this._startTime = value; }
        }

        public String EndTime
        {
            get { return this._endTime; }
            set { this._endTime = value; }
        }

        public String Iwon
        {
            get { return this._iwon; }
            set { this._iwon = value; }
        }

        public String AddIwon
        {
            get { return this._addIwon; }
            set { this._addIwon = value; }
        }

        public String OutHospSlot
        {
            get { return this._outHospSlot; }
            set { this._outHospSlot = value; }
        }

        public String DataRowState
        {
            get { return this._dataRowState; }
            set { this._dataRowState = value; }
        }

        public SCH3001U00GrdSCH3000Info() { }

        public SCH3001U00GrdSCH3000Info(String jukyongDate, String jundalTable, String jundalPart, String gumsaja, String yoilGubun, String startTime, String endTime, String iwon, String addIwon, String outHospSlot, String dataRowState)
        {
            this._jukyongDate = jukyongDate;
            this._jundalTable = jundalTable;
            this._jundalPart = jundalPart;
            this._gumsaja = gumsaja;
            this._yoilGubun = yoilGubun;
            this._startTime = startTime;
            this._endTime = endTime;
            this._iwon = iwon;
            this._addIwon = addIwon;
            this._outHospSlot = outHospSlot;
            this._dataRowState = dataRowState;
        }

    }
}